ontology/dprod/dprod-ontology.ttl

```diff
@@ -1,160 +1,95 @@
 # baseURI: https://ekgf.github.io/dprod/
-# imports: http://purl.org/dc/terms/
-# imports: http://www.w3.org/2004/02/skos/core
-# imports: http://www.w3.org/ns/prov-o#
-# imports: http://www.w3.org/ns/dcat
+# imports: http://www.w3.org/ns/dcat#
 # prefix: dprod
 
-
-@base: <https://ekgf.github.io/dprod/#> .
-@prefix dprod: <https://ekgf.github.io/dprod/#> .
+@base <https://ekgf.github.io/dprod/> .
+@prefix dprod: <https://ekgf.github.io/dprod/> .
 @prefix dcat: <http://www.w3.org/ns/dcat#> .
 @prefix dct: <http://purl.org/dc/terms/> .
-@prefix dc: <http://purl.org/dc/elements/1.1/> .
-@prefix dctype: <http://purl.org/dc/dcmitype/> .
-@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix prov: <http://www.w3.org/ns/prov#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
-@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+# Since Dublin Core is not an ontology, declare here that the properties used are to be treated as AnnotationProperties in OWL
+dct:description a owl:AnnotationProperty .
+dct:contributor a owl:AnnotationProperty .
+dct:license a owl:AnnotationProperty .
+dct:modified a owl:AnnotationProperty .
 
 dprod:
```

Comment:
Why is the dprod ontology here? That's supposed to just be in `dprod-ontology.ttl`. Both files are loaded by the specsite generator into one graph and then processed.